client/src/layout/Scroller.js

```diff
@@ -15,21 +15,6 @@ const Scroller = ({
   mentionDatas,
   setMentionDatas,
```

Comment:
Unused props and imports, make sure to take them out!

Reply:
Took care of it. Thx !